lib/Frontend/CompilerInvocation.cpp

```diff
@@ -34,7 +34,7 @@ using namespace llvm::opt;
 /// The path for Swift libraries in the OS on Darwin.
 #define DARWIN_OS_LIBRARY_PATH "/usr/lib/swift"
 
-static constexpr const char *const supportedLocaleCodes[] = {
+static constexpr const char *const localeCodes[] = {
```

Comment:
This doesn't make sense to be "supportedLocaleCodes" now. So I changed it to only be `localeCodes`.